src/Settings.js

```diff
@@ -169,6 +169,11 @@ export class Settings {
             return false
         }
 
+        // If it's an email field and Email Protection is enabled, return true regardless of other options
+        if (subtype === 'emailAddress' && this.featureToggles.emailProtection && this.availableInputTypes.email) {
+            return true
+        }
+
```

Comment:
During Android integration I figured that this should be moved before all other checks otherwise we'd be making an extra call to `getAvailableInputTypes` for email fields if `identities: undefined`.